packages/constants/src/constant.ts

```diff
@@ -0,0 +1,249 @@
+function Key(id: string, label: string, value: string, url: string) {
```

Comment:
Since this is now a .ts file, do we want to make this an `interface` instead?

Reply:
added type